core/renderers/geras/renderer.js

```diff
@@ -59,3 +60,16 @@ Blockly.geras.Renderer.prototype.init = function() {
   Blockly.geras.Renderer.superClass_.init.call(this);
   this.highlightConstants = new Blockly.geras.HighlightConstantProvider();
 };
+
+/**
+ * Initialize the renderer.  Geras has a highlight provider in addition to
+ * the normal constant provider.
+ * @return {Blockly.geras.HighlightConstantProvider} The highlight constant
+ *     provider.
+ * @package
+ */
+Blockly.geras.Renderer.prototype.getHighlightConstants = function() {
+  return Blockly.blockRendering.renderer.highlightConstants;
```

Comment:
should probably use this.highlightConstants instead of getting the current renderer which either is this one or may be incompatible.